tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -105,7 +105,12 @@ const shareFileFolder = function (
  * @param {string} data.expireDate Expiry date of the share
  */
 const createPublicLink = function (sharer, data) {
-  const { path, permissions = 'read', name, password, expireDate } = data
+  let { path, permissions = 'read', name, password, expireDate } = data
+
+  if (typeof expireDate !== 'undefined') {
```

Comment:
I think we can remove this `if` check and just keep the body of it.
```suggestion
  if (typeof expireDate !== 'undefined') {
```

Reply:
`calculateDate` expects a string to do some parsing on, so it breaks if its given `undefined`, could place the if statement there, but that does not feel right because that function cannot do anything with `undefined`